File: archive_legacy/backend/check_cols.js
require('dotenv').config();
const sql = require('./db.js');

async function run() {
  const result = await sql`SELECT column_name FROM information_schema.columns WHERE table_name = 'customers'`;
  console.log('Columns in customers table:', result.map(r => r.column_name));
  
  process.exit(0);
}

run();

File: backend/check_cols.py
import os
import pandas as pd
from sqlalchemy import create_engine
from dotenv import load_dotenv
load_dotenv()
engine = create_engine(os.getenv('DATABASE_URL').replace('?pgbouncer=true',''))
df = pd.read_sql("SELECT column_name FROM information_schema.columns WHERE table_name = 'raw_items_master'", engine)
print(df.to_string())
